spec/features/post/index.html.erb_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe 'Post index page', type: :feature do
+  describe 'test' do
+    before :each do
+      # @user = User.create(name: 'Tom', bio: 'Teacher from Mexico.', photo: 'https://unsplash.com/photos/F_-0BxGuVvo')
+      # @id = @user.id
+      # @first_post = Post.create(user_id: @user, title: 'Hello', text: 'Make a new comment on this post')
+      # @second_post = Post.create(user_id: @user, title: 'hello', text: 'Make a new comment on this post')
+      # @comment = Comment.create(post_id: @second_post, author: @user, text: ' hello world!')
+      visit '/users/83/posts'
+    end
+
+    it 'show the username' do
+      expect(page).to have_content('Tom')
+    end
+
+    it 'show the title' do
+      expect(page).to have_content('Post information')
+    end
+
+    it 'show the user photo' do
+      expect(page.body).to have_content('https://unsplash.com/photos/F_-0BxGuVvo')
+    end
+
+    it 'show the number of post' do
+      expect(page.body).to have_content('Number of post: 1')
+    end
+
+    it 'show the post title' do
+      expect(page).to have_content('Hello')
+    end
+
+    it 'show the number of comments' do
+      expect(page).to have_content('Comments: 1')
+    end
```

Comment:
- [ ] You need to have two tests about the comments and number of comments, this can be counted as one, but please add another one
- Test that asserts you can see the first comments on a post.
- Test that asserts you can see how many comments a post has.

Reply:
- [ ] as previously requested kindly add a test to assert you can see the first comment on a post and also see how many comments a post has below is a screenshot of all tests required for thiss view
![image](https://user-images.githubusercontent.com/61408860/203869178-ddb70889-52ed-471a-81e5-47c8ced8e6a1.png)